RELEASE_PROCESS.md

```diff
@@ -0,0 +1,151 @@
+# Releasing InSpec
+
+## Promote Dependency Gems
+
+It's important that dependency gems be promoted (released to rubygems.org) BEFORE the last PR you intend to release is merged into inspec/inspec. This is because during the inspec merge process, software artifacts are created that embed the dependencies (such as omnibus packages, Habitat packages, and Docker images). During promotion, those artifacts are simply promoted, not rebuilt. If you need a later version of one of these dependencies and have no legitimate PR to merge, a dummy PR may be needed.
+
+### Key Dependencies
+
+Any change to these gems must be promoted to rubygems.org prior to releasing InSpec.
+
+#### train / train-core
+
+Train is the connectivity library for InSpec, and is the most critical dependency of InSpec.
+
+train-core is a stripped-down version of train, with no cloud provider plugins and no gems that require compilation during installation.
+
+You can promote train by running, in #inspec-notify,
+```
+/expeditor promote inspec/train:master 3.2.23
+```
+
+Note that the version you give to Expeditor is the plain version (not the tag, which includes a `v` prefix).
+
+#### train-aws, train-winrm, and train-habitat
```

Comment:
Do you promote these gems in the same way as you promote inspec/train above?

Reply:
Yes, added.